bundles/binding/org.openhab.binding.lcn/readme_conf.txt

```diff
@@ -0,0 +1,48 @@
+﻿openHAB LCN binding: "openhab.cfg" Configuration Guide
+Author: Tobias Jüttner
+Date: 07/28/2015
+
+The "openHAB LCN binding" connects to one or more LCN-PCHK instances via TCP/IP.
+*** This means 1 unused LCN-PCHK license is required! ***
+The minimum recommended version is LCN-PCHK 2.8 (older versions will also work, but lack some functionality).
+Visit www.lcn.de for updates.
+
+
+Example configuration for a connection
+--------------------------------------
+
+lcn:id1=myhome
+lcn:address1=localhost:4114
+lcn:username1=lcn
+lcn:password1=test123
+lcn:mode1=native200
+
+This defines a connnection (#1) to LCN-PCHK on "localhost" port 4114.
+"myhome" will be the connection's unique identifier ("connid", used in openHAB binding-definitions).
+"address" can be any IP or host name. The port (":4114") is optional.
+
+"mode" specifies the operation mode for rhe LCN bus:
+- LCN busses with SOLELY modules with firmware 170206 (feb. 2013) or newer use "native200"
+  (means: output-ports have 200 dimming steps)
+- All other LCN busses use "native50"
+  (means: output-ports have 50 dimming steps)
+This is an important setting as the operation mode is programmed into the LCN modules by LCN-PRO.
+If you experience unexpected behavior while dimming output-ports, the connection is probably in the wrong operation mode!
+*** For older versions than LCN-PCHK 2.8: Use "percent50" and "percent200" instead (loses some precision). ***
+
+
+Adding a SECOND connection (#2)
+-------------------------------
+
+lcn:id2=myotherhome
+lcn:address2=...
+...
+
+More connections are defined by incrementing the appended counter ("2" in the example).
+*** Don't skip any numbers! GOOD: 1,2,3,4 BAD: 1,3,4 ***
```

Comment:
Isn't it a best practice for openHAB to support multiple external systems? It is actually a limitation of the KNX binding that it only supports a single connection to an KNX interface.